File: app/src/main/java/com/example/whattowatch/model/mappers/PersonMapper.java
package com.example.whattowatch.model.mappers;

import com.example.whattowatch.model.mymodel.MyPersonModel;
import com.example.whattowatch.model.response.person.Person;

public class PersonMapper {

    public static MyPersonModel convertPerson(Person person){
        return new MyPersonModel(
                person.getBirthday(),
                person.getKnownForDepartment(),
                person.getId(),
                person.getName(),
                person.getGender(),
                person.getBiography(),
                person.getPopularity(),
                person.getPlaceOfBirth(),
                person.getProfilePath()
        );
    }
}

File: app/src/main/java/com/example/whattowatch/model/mappers/PersonMapper.kt
package com.example.whattowatch.model.mappers

import com.example.whattowatch.model.mymodel.MyPersonModel
import com.example.whattowatch.model.response.person.Person

object PersonMapper {
    fun convertPerson(person: Person): MyPersonModel {
        return MyPersonModel(
                person.birthday,
                person.knownForDepartment,
                person.id,
                person.name,
                person.gender,
                person.biography,
                person.popularity,
                person.placeOfBirth,
                person.profilePath
        )
    }
}